### PR TITLE
fix focus trap in navigation

### PR DESCRIPTION
**Pull request by felixhaeberle**
Fix issue described in https://www.drupal.org/project/olivero/issues/3093455

### CODE REVIEW

**Comment by felixhaeberle** [comment]
thanks, makes sense! will do that today

**Comment by felixhaeberle** [comment]
For performance reason, I considered `offsetParent` first.
It's 70% faster. https://jsperf.com/check-hidden

File: src/js/navigation.js [hunk 1]
```diff
@@ -62,12 +62,12 @@
   mobileNavWrapper.addEventListener('keydown', function(e) {
     if (e.key === 'Tab' || e.keyCode === 9) {
       if ( e.shiftKey ) /* shift + tab */ {
-        if (document.activeElement === firstFocusableEl) {
+        if (document.activeElement === firstFocusableEl && isDesktopNav() === false) {
```
Comment (mherchel) [inline src/js/navigation.js]: Just a minor code style thing... but you could simplify this to `(document.activeElement === firstFocusableEl && !isDesktopNav())`

Comment (felixhaeberle) [reply src/js/navigation.js]: 👍

**Pull request merged**